Boyer_Moore_Algorithm/Boyer_Moore.cs

```diff
@@ -0,0 +1,115 @@
+/**
+    Code contributed by Hardev Khandhar for GirlScript Summer of Code 2020
+    link: https://github.com/HardevKhandhar
+*/
+
```

Comment:
No need this line @HardevKhandhar . it can track through git log. shall we remove this?